packages/dds/tree/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -17,17 +17,17 @@ const tag1: RevisionTag = brand(1);
 const tag2: RevisionTag = brand(2);
 
 function compose(changes: TestChangeset[]): TestChangeset {
-    changes.forEach(deepFreeze);
+    changes.forEach((c) => deepFreeze(c));
```

Comment:
Is this change necessary?

Reply:
It is: `forEach` takes a delegate that takes 3 parameters, whose types do not match `deepFreeze`.